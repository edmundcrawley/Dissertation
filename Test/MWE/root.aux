\relax 
\@input{chapter0.aux}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}This is Chapter 1}{1}}
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{lof}{\contentsline {figure}{\numberline {1.1}{\ignorespaces Time Aggregation Induces Serial Correlation\relax }}{2}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{fig:TimeAggExample}{{1.1}{2}}
\@writefile{lot}{\contentsline {table}{\numberline {1.1}{\ignorespaces Minimum-Distance Partial Insurance and Variance Estimates\relax }}{4}}
\newlabel{table:ReplicationTable}{{1.1}{4}}
