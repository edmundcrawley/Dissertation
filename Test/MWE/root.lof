\addvspace {10\p@ }
\contentsline {figure}{\numberline {1.1}{\ignorespaces Time Aggregation Induces Serial Correlation\relax }}{2}
