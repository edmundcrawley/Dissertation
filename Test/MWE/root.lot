\addvspace {10\p@ }
\contentsline {table}{\numberline {1.1}{\ignorespaces Minimum-Distance Partial Insurance and Variance Estimates\relax }}{4}
