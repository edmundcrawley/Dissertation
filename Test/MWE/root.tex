\documentclass[12pt,oneside,final]{thesis}

\usepackage{graphicx}

\usepackage{fancyhdr}    % Use nice looking headers along with the required footer page numbers 



\usepackage[font=singlespacing, labelfont=bf]{caption}
%\usepackage{setspace}
\newlength\TableWidth
\setlength{\paperheight}{11in}
\setlength{\paperwidth}{8.5in}

%Define the header/footer style
\pagestyle{fancy}
\fancyhf{}
\setlength{\headheight}{15pt}
\lhead{\leftmark}
\cfoot{\thepage}
\renewcommand{\headrulewidth}{0pt}
\fancypagestyle{plain}{% Redefine ``plain'' style for chapter boundaries
	\fancyhf{} % clear all header and footer fields
	\fancyfoot[C]{\thepage} % except the center
	\renewcommand{\headrulewidth}{0pt}
	\renewcommand{\footrulewidth}{0pt}}

\begin{document}

\title{ESSAYS ON CONSUMPTION}
\author{Edmund Crawley}
\degreemonth{May}
\degreeyear{2018} 
\dissertation
\doctorphilosophy
\copyrightnotice


% add your chapters, best way is to have separate TeX files for each chapter
\include{chapter0}
\chapter{This is Chapter 1}
\begin{figure}
	\caption{Time Aggregation Induces Serial Correlation}
	\label{fig:TimeAggExample}
	\includegraphics[width=1\textwidth]{TimeAggExample.pdf}
\end{figure}

The first two columns of table \ref{table:Persistence} show that the degree of persistence in the original BPP model makes a big difference to the estimate of $\phi$, while all of the time aggregation models show similar estimates for $\psi$. This suggests the difference we see in the estimates of $\phi$ between BPP original model and the time aggregated model may be driven, at least in part, by misspecification in the model of transitive income shocks. It is reassuring that, in contrast to the BPP model, the values of both $\phi$ and $\psi$ are relatively robust to the exact specification of transitive persistence in the time aggregated model.
The first two columns of table \ref{table:Persistence} show that the degree of persistence in the original BPP model makes a big difference to the estimate of $\phi$, while all of the time aggregation models show similar estimates for $\psi$. This suggests the difference we see in the estimates of $\phi$ between BPP original model and the time aggregated model may be driven, at least in part, by misspecification in the model of transitive income shocks. It is reassuring that, in contrast to the BPP model, the values of both $\phi$ and $\psi$ are relatively robust to the exact specification of transitive persistence in the time aggregated model.
The first two columns of table \ref{table:Persistence} show that the degree of persistence in the original BPP model makes a big difference to the estimate of $\phi$, while all of the time aggregation models show similar estimates for $\psi$. This suggests the difference we see in the estimates of $\phi$ between BPP original model and the time aggregated model may be driven, at least in part, by misspecification in the model of transitive income shocks. It is reassuring that, in contrast to the BPP model, the values of both $\phi$ and $\psi$ are relatively robust to the exact specification of transitive persistence in the time aggregated model.\\
The first two columns of table \ref{table:Persistence} show that the degree of persistence in the original BPP model makes a big difference to the estimate of $\phi$, while all of the time aggregation models show similar estimates for $\psi$. This suggests the difference we see in the estimates of $\phi$ between BPP original model and the time aggregated model may be driven, at least in part, by misspecification in the model of transitive income shocks. It is reassuring that, in contrast to the BPP model, the values of both $\phi$ and $\psi$ are relatively robust to the exact specification of transitive persistence in the time aggregated model.

\input RepTable6.tex

The first two columns of table \ref{table:Persistence} show that the degree of persistence in the original BPP model makes a big difference to the estimate of $\phi$, while all of the time aggregation models show similar estimates for $\psi$. This suggests the difference we see in the estimates of $\phi$ between BPP original model and the time aggregated model may be driven, at least in part, by misspecification in the model of transitive income shocks. It is reassuring that, in contrast to the BPP model, the values of both $\phi$ and $\psi$ are relatively robust to the exact specification of transitive persistence in the time aggregated model.
The first two columns of table \ref{table:Persistence} show that the degree of persistence in the original BPP model makes a big difference to the estimate of $\phi$, while all of the time aggregation models show similar estimates for $\psi$. This suggests the difference we see in the estimates of $\phi$ between BPP original model and the time aggregated model may be driven, at least in part, by misspecification in the model of transitive income shocks. It is reassuring that, in contrast to the BPP model, the values of both $\phi$ and $\psi$ are relatively robust to the exact specification of transitive persistence in the time aggregated model.
The first two columns of table \ref{table:Persistence} show that the degree of persistence in the original BPP model makes a big difference to the estimate of $\phi$, while all of the time aggregation models show similar estimates for $\psi$. This suggests the difference we see in the estimates of $\phi$ between BPP original model and the time aggregated model may be driven, at least in part, by misspecification in the model of transitive income shocks. It is reassuring that, in contrast to the BPP model, the values of both $\phi$ and $\psi$ are relatively robust to the exact specification of transitive persistence in the time aggregated model.
The first two columns of table \ref{table:Persistence} show that the degree of persistence in the original BPP model makes a big difference to the estimate of $\phi$, while all of the time aggregation models show similar estimates for $\psi$. This suggests the difference we see in the estimates of $\phi$ between BPP original model and the time aggregated model may be driven, at least in part, by misspecification in the model of transitive income shocks. It is reassuring that, in contrast to the BPP model, the values of both $\phi$ and $\psi$ are relatively robust to the exact specification of transitive persistence in the time aggregated model.
The first two columns of table \ref{table:Persistence} show that the degree of persistence in the original BPP model makes a big difference to the estimate of $\phi$, while all of the time aggregation models show similar estimates for $\psi$. This suggests the difference we see in the estimates of $\phi$ between BPP original model and the time aggregated model may be driven, at least in part, by misspecification in the model of transitive income shocks. It is reassuring that, in contrast to the BPP model, the values of both $\phi$ and $\psi$ are relatively robust to the exact specification of transitive persistence in the time aggregated model.

%\input C:/Users/edmun/OneDrive/Documents/Research/Dissertation/Chapter1/Tables/RepTable6.tex
%\include{chapter2}
%\include{chapter3}
%\include{appendix}

%% REFERENCES

% if you use BIBTEX
%\bibliographystyle{econometrica}
%\bibliographystyle{IEEEtran}
%\bibliography{Reference_new}


%\begin{vita}
%
%\begin{wrapfigure}{l}{0pt}
%\includegraphics[width=2in,height=2.5in,clip,keepaspectratio]{rjvheadshot}
%\end{wrapfigure}
%
%\end{vita}
\end{document}
