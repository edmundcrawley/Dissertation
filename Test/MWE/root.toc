\contentsline {chapter}{Abstract}{ii}
\contentsline {chapter}{Acknowledgments}{iii}
\contentsline {chapter}{List of Tables}{vi}
\contentsline {chapter}{List of Figures}{vii}
\contentsline {chapter}{\numberline {1}This is Chapter 1}{1}
