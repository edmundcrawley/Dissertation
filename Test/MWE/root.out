\BOOKMARK [0][-]{chapter*.1}{Abstract}{}% 1
\BOOKMARK [0][-]{chapter*.2}{Acknowledgments}{}% 2
\BOOKMARK [0][-]{chapter*.5}{List of Tables}{}% 3
\BOOKMARK [0][-]{chapter*.6}{List of Figures}{}% 4
\BOOKMARK [0][-]{chapter.1}{Time Aggregation in Panel Data on Income and Consumption}{}% 5
\BOOKMARK [1][-]{section.1.1}{Abstract}{chapter.1}% 6
\BOOKMARK [1][-]{section.1.2}{Introduction}{chapter.1}% 7
\BOOKMARK [2][-]{subsection.1.2.1}{What is Time Aggregation?}{section.1.2}% 8
